buf/registry/owner/v1beta1/user_service.proto

```diff
@@ -107,7 +107,8 @@ message CreateUsersRequest {
     // The configurable URL that represents the homepage for a User.
     string url = 4 [
       (buf.validate.field).string.uri = true,
-      (buf.validate.field).string.max_len = 255
+      (buf.validate.field).string.max_len = 255,
+      (buf.validate.field).ignore_empty = true
```

Comment:
Curious, why does this have to be empty? There is no `min_len` set on the field?

Reply:
Because otherwise the uri validator fails: it does not accept an empty value as a valid URI.

Reply:
Got it! That makes sense.

Reply:
An empty string isn't a valid URI so fails the string on string uri: `values[0].url: value must be a valid URI [string.uri]`.